src/AutoRest.CSharp/MgmtTest/Generation/ResourceCollectionTestWriter.cs

```diff
@@ -129,7 +129,12 @@ public void WriteGetCollection(MgmtTypeProvider parentTp, string requestPath, Ex
                     }
                 case MgmtExtensions extension:
                     {
-                        _writer.Append($"var collection = GetArmClient().Get{extension.ArmCoreType.Name}(new {typeof(Azure.Core.ResourceIdentifier)}({MgmtBaseTestWriter.FormatResourceId(realRequestPath):L}))");
+                        if (extension == MgmtContext.Library.TenantExtensions) {
+                            _writer.Append($"var collection = GetArmClient().GetTenants().GetAll().GetEnumerator().Current");
```

Comment:
```suggestion
                            _writer.Append($"var collection = GetArmClient().GetTenants().First()");
```
And you might need to manually add the `System.Linq` namespace here